datastore/h2gis/src/test/groovy/org/orbisgis/datastore/h2gis/H2GISTest.groovy

```diff
@@ -59,4 +62,67 @@ class H2GISTest {
         assert 100 == ds.fetchSize
         assert 50 == ds.batchInsertSize
     }
+
+    @Test
+    void openH2GISFromPath() {
+        def ds = H2GIS.open("./target/db_h2gis")
+        assert ds
+    }
```

Comment:
Can you add `assert new File("./target/db_h2gis.mv.db").exists()`  to check that the db file exists.